cmd/backup-manager/app/backup/manager.go

```diff
@@ -316,6 +316,27 @@ func (bm *Manager) performBackup(ctx context.Context, backup *v1alpha1.Backup, d
 	// run br binary to do the real job
 	backupErr := bm.backupData(ctx, backup, bm.StatusUpdater)
 
+	defer func() {
+		// Calculate the backup size for ebs backup job even if it fails
+		if bm.Mode == string(v1alpha1.BackupModeVolumeSnapshot) && !bm.Initialize {
+			backupSize, err := util.CalcVolSnapBackupSize(ctx, backup.Spec.StorageProvider)
+			if err != nil {
+				klog.Warningf("Failed to calc volume snapshot backup size %d bytes, %v", backupSize, err)
+				return
+			}
+
+			backupSizeReadable := humanize.Bytes(uint64(backupSize))
+
+			updateStatus := &controller.BackupUpdateStatus{
+				BackupSize:         &backupSize,
+				BackupSizeReadable: &backupSizeReadable,
+			}
+
+			bm.StatusUpdater.Update(backup, nil,
```

Comment:
How about logging the result of `Update`?

Reply:
The possible error of `update` is already handled by `realBackupConditionUpdater.Update`